features/account/registering.feature

```diff
@@ -0,0 +1,33 @@
+@customer_account
+Feature: Account registration
```

Comment:
maybe `user/account` would be good path? But we use a folder name as a tag name currently. Can we keep this convention?